### ISSUE TITLE
Rethink drop-down menu, particularly chevrons

### ISSUE THREAD

**Issue by mor10** [wprig/wprig#1]
## Issue Overview
Drop-down menu doesn't work quire right. Chevron shows up where it shouldn't, has minimal functionality. Could use better CSS and possibly retire the whole JS solution.

**Comment by mor10** [comment]
My idea:

Use `focus-in` for mouse and keyboard nav (as is the case now) and use the chevrons _only_ for touch-interactions. Possibly apply the `screen-reader-text` class to the chevron buttons so they are skipped over by keyboard users since they don't do anything. 

Doing this would eliminate the need for most of the JS and leave us with a very simple function. No need for the string passing and other complex behavior.

**Comment by mor10** [comment]
closed via pull request above. good work @bamadesigner
